### PR TITLE
aaronschultz47

### PR DESCRIPTION
**Pull request by aaronschultz47**
Added LTC6603 and its first DC1304A-B demo board.

### CODE REVIEW

**Review by malavikasajikumar — DISMISSED** [review]
A few documentation changes needed and a function that was not implemented in the example code.

File: LTSketchbook/Part Number/6000/6603/DC1304A_B/DC1304A_B.ino [hunk 1]
```diff
@@ -0,0 +1,363 @@
+
+
+/*!
+Linear Technology DC1304A-B Demonstration Board.
+DC1304: Dual Matched, High Frequency Bandpass/Lowpass Filters
```
Comment (malavikasajikumar) [inline LTSketchbook/Part Number/6000/6603/DC1304A_B/DC1304A_B.ino]: Replace the demo board number with part number.

**Review by malavikasajikumar — CHANGES_REQUESTED** [review]
Single documentation change requested as a trial.

File: LTSketchbook/Part Number/6000/6603/DC1304A_B/DC1304A_B.ino [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+
+@verbatim
+1304
```
Comment (malavikasajikumar) [inline LTSketchbook/Part Number/6000/6603/DC1304A_B/DC1304A_B.ino]: Remove this line.

File: LTSketchbook/Part Number/6000/6603/DC1304A_B/DC1304A_B.ino [hunk 3]
```diff
@@ -0,0 +1,363 @@
...
+
+The Linear Technology Linduino is not affiliated with the official Arduino team.
+However, the Linduino is only possible because of the Arduino team's commitment
+to the open-source community.  Please, visit http://www.arduino.cc and
+http://store.arduino.cc , and consider a purchase that will help fund their
+ongoing work.
+ */
+
+/*! @file
+    @ingroup DC1304
```
Comment (malavikasajikumar) [inline LTSketchbook/Part Number/6000/6603/DC1304A_B/DC1304A_B.ino]: The group should be LTC6603.

File: LTSketchbook/Part Number/6000/6603/DC1304A_B/DC1304A_B.ino [hunk 4]
```diff
@@ -0,0 +1,363 @@
...
+  Serial.print(F("* The LTC6603 is turning on *\n"));
+  control_byte = (control_byte & LTC6603_ONOFF_MASK) | LTC6603_PRUP;
+  spi_out = control_byte;
+  LTC6603_write(LTC6603_CS, &spi_out, (uint8_t)1);
+  Serial.print ("\Control byte 0x" + String(control_byte, HEX) + "\n");
+
+}
+
+void menu_5_changeclockfrequency()
+{
```
Comment (malavikasajikumar) [inline LTSketchbook/Part Number/6000/6603/DC1304A_B/DC1304A_B.ino]: This function is not implemented.

File: LTSketchbook/Part Number/6000/6603/DC1304A_B/DC1304A_B.ino [hunk 5]
```diff
@@ -0,0 +1,363 @@
...
+void  menu_1_filter_settings()
+{
+  int16_t user_command; // The user input command
+  while (1)
+  {
+    Serial.print(F("*************************\n\n"));
+    Serial.println (F("0-Enter External Frequency"));
+    Serial.println (F("1-Set Gain"));
+    Serial.println (F("2-Set Lowpass"));
+    Serial.println (F("3-Set No Function Code"));
```
Comment (malavikasajikumar) [inline LTSketchbook/Part Number/6000/6603/DC1304A_B/DC1304A_B.ino:165]: Can these menu options be printed from the print_prompt(). If not, print_prompt() can be removed.

File: LTSketchbook/Part Number/6000/6603/DC1304A_B/DC1304A_B.ino [hunk 6]
```diff
@@ -0,0 +1,363 @@
...
+  Serial.print(F("* This program demonstrates how to send data to the LTC6603.    *\n"));
+  Serial.print(F("*                                                               *\n"));
+  Serial.print(F("* Set the baud rate to 115200 and select the newline terminator.*\n"));
+  Serial.print(F("*                                                               *\n"));
+  Serial.print(F("*****************************************************************\n"));
+}
+
+//! Prints main menu.
+void print_prompt()
+{
```
Comment (malavikasajikumar) [inline LTSketchbook/Part Number/6000/6603/DC1304A_B/DC1304A_B.ino:335]: These menu options are different from the menu options printed from loop()

Comment (aaronschultz47) [reply LTSketchbook/Part Number/6000/6603/DC1304A_B/DC1304A_B.ino:335]: It's ok.  this is the very first thing that shows up, only once.

File: LTSketchbook/Part Number/6000/6603/DC1304A_B/DC1304A_B.ino (hunk 5 above)
Comment (aaronschultz47) [reply LTSketchbook/Part Number/6000/6603/DC1304A_B/DC1304A_B.ino:165]: It's ok.  this is different from the general print_prompt.  This is only for menu item 1.

**Pull request merged**